application/src/main/java/org/togetherjava/tjbot/commands/MessageReceiver.java

```diff
@@ -46,4 +56,13 @@ public interface MessageReceiver extends Feature {
      *        message that was edited
      */
     void onMessageUpdated(MessageUpdateEvent event);
+
+    /**
+     * Triggered by the core system whenever an existing message was deleted in a text channel of a
```

Comment:
"of [**a**] guild"

Reply:
what?

Reply:
ur text says "in a text channel of guild the bot", its missing an "a" before the "guild".

Reply:
actually. nvm. its there, it just wasnt really visible in the github ui.